src/drc/api/views/bestandsdeel.py

```diff
@@ -9,12 +12,42 @@
 from drc.datamodel.models.bestandsdeel import BestandsDeel
 
 
+@extend_schema_view(
+    update=extend_schema(
+        summary=_("Upload een bestandsdeel."),
+    ),
+)
+@extend_schema(
+    responses={
+        200: inline_serializer(
+            name="BestandsDeelResponse",
+            fields={
+                "url": serializers.HyperlinkedIdentityField(
+                    view_name="bestandsdeel_update"
+                ),
+                "lock": serializers.CharField(
+                    help_text=BestandsDeelSerializer._declared_fields["lock"].help_text,
+                ),
+                "omvang": serializers.IntegerField(
+                    help_text=BestandsDeel.omvang.field.help_text, required=False
+                ),
+                "inhoud": serializers.URLField(
+                    help_text=BestandsDeel.inhoud.field.help_text, required=False
+                ),
+                "voltooid": serializers.BooleanField(
+                    help_text=BestandsDeelSerializer.Meta.extra_kwargs["voltooid"][
+                        "help_text"
+                    ],
+                    required=False,
+                ),
+                "volgnummer": serializers.IntegerField(
+                    help_text=BestandsDeel.volgnummer.field.help_text, required=False
+                ),
+            },
+        )
+    }
+)
```

Comment:
Deze `extend_schema` decorator roep je in de decorator hierboven ook aan, ik zou deze ook nog even samenvoegen (in plaats van twee decorators, een decorator gebruiken)

Reply:
[Die zijn al samengevoegd](https://github.com/VNG-Realisatie/documenten-api/pull/178/files/8cf0d09bcd2672217448fcff043947d760d5dee0#diff-68de1d00016ae451b272e5e72633b13b74ce48ed5847f388c84cb4f3abab15ecR16), vandaar m'n oorspronkelijke comment ;)

Reply:
ah die had ik even gemist